src/features/embalm/stepContent/hooks/useArchaeologistList.ts

```diff
@@ -35,7 +36,7 @@ export function useArchaeologistList() {
     // Or their free bond is less than their digging fee
     const archIsVisible =
       resurrectionTimeMs - Date.now() < maxRewrapIntervalMs &&
-      a.profile.minimumDiggingFee.lte(a.profile.freeBond);
+      a.profile.minimumDiggingFeePerSecond.lte(a.profile.freeBond);
```

Comment:
Just commenting so it's not forgotten -- this comparison will likely need to be updated to use the time-based curse estimate instead

Reply:
Thanks for pointing that out.